policies/rules.md

```diff
@@ -0,0 +1,163 @@
+
+Projects compete in four problem areas, and are evaluated on effectiveness, accessibility, and impact.
+
+### The Problems
+
+We're passionate about improving the quality of life for people during and after the COVID-19 Pandemic by addressing the following problems:
+
+1. Supporting Crisis Response
+2. Understanding The Pandemic
+3. Social Distancing & Isolation 
+4. Scarcity & The Economy
+
+#### Supporting Crisis Response
+
+First responders, healthcare workers and emergency response managers are on the front lines of the pandemic. How can we help to work more effectively, safely, and quickly?
+
+Examples of apps in this category might include services to crowd source reporting of crisis information, using machine learning and data science to quickly identify problems to be addressed, or allow more efficient distribution of health care and emergency resources.
+
+#### Understanding The Pandemic
+
+How can we better understand the current state of the pandemic and make informed decisions about the future? How do we evaluate and trust the information around us?
+
+An app in this category might help provide verified answers to questions about COVID-19 and the pandemic, predict if the curve is being flattened, or help people report and filter out mis-information.
+
+#### Social Distancing & Isolation
+
+From working at home, assisting the elderly and at need, to entertaining and educating our children few parts of the pandemic more broadly impact society than the Social Isolation and Distancing we're all practicing.
+
+How do we limit the impact and costs associated with Social Isolation and Distancing, and ensure that we're ready to pick up where we left off when we come back together?
+
+Apps in this category might help connect people together, identify people in distress, facilitate remote working, on assist parents and educators continue classes.
+
+#### Scarcity & The Economy
+
+Economics is the study of scarcity, and during the pandemic it's a struggle to understand the situation and how to most effectively use limited resources to maximum effect.
+
+How we better understand and manage the economic impacts, and use our resources most effectively to weather the largest economic change in modern history.
+
+App in this category could include helping understand and manage changes in the work force, evaluate and track the effectiveness of government interventions, or better distribute scarce resources.
+
+### Your Solution
+
+We believe faster, more effective and more available solutions are preferred to slower, less useful and limited ones.
+
+Therefore we will be judging projects on three main criteria:
+
+1. **Effectiveness** - How well does this project address the problem it's attempting to solve?
+2. **Accessibility** - How easily are people are able to access this project?
+3. **Impact** - How many people have benefitted from this project?
```

Comment:
Just nit-picking but I think impact should go beyond 'number of people' (assuming the team can demonstrate what kind of impact a project can have e.g.  general economic, social, safety, etc.)

Reply:
I moved this into #wg-policy, I do agree this needs some thought and conversation around it, but I also want participants to have these rules out as-is.